include/cinder/ConcurrentCircularBuffer.h

```diff
@@ -107,7 +107,13 @@ class ConcurrentCircularBuffer : private Noncopyable {
 	}
 	
 	//! Returns the number of items the buffer can hold
-	size_t size() const { return (size_t)mContainer.capacity(); }
+	size_t getCapacity() const { return (size_t)mContainer.capacity(); }
+	
+	//! Returns the number of items the buffer is currently holding
+	size_t getSize() {
```

Comment:
I think this should be const, and as such I don't believe it needs the protection of the mutex (see: [you don't know const](http://herbsutter.com/2013/01/01/video-you-dont-know-const-and-mutable/)).